packages/pegasus/src/pegasus.js

```diff
@@ -498,16 +498,9 @@ const makePegasus = (zcf, board, namesByAddress) => {
         winner,
       ) => {
         // Transfer the amount to our backing seat.
-        const currentLoser = loser.getAmountAllocated(loserKeyword, localBrand);
-        const currentWinner = winner.getAmountAllocated(
-          winnerKeyword,
-          localBrand,
-        );
-        const newLoser = AmountMath.subtract(currentLoser, amount);
-        const newWinner = AmountMath.add(currentWinner, amount);
-        const loserStage = loser.stage({ [loserKeyword]: newLoser });
-        const winnerStage = winner.stage({ [winnerKeyword]: newWinner });
-        zcf.reallocate(loserStage, winnerStage);
+        winner.incrementBy({ [winnerKeyword]: amount });
+        loser.decrementBy({ [loserKeyword]: amount });
```

Comment:
There is no actual safety issue here. But, for the sake of developing habits that help preserve safety elsewhere, when there is no reason not to, I prefer implementing transfers by always doing losses before gains, as the old code did.

```suggestion
        loser.decrementBy({ [loserKeyword]: amount });
        winner.incrementBy({ [winnerKeyword]: amount });
```

Reply:
Done!